RELEASE_NOTES.md

```diff
@@ -19,7 +19,56 @@
 
 Version v0.2.0 -- Release date 2020-06-20
 ==========================================
-Lots of stuff to fill in...
+* Extensive updates to Data Repository.
+* Discretization:
+  * Finite element interface
+    * Kernel launching interface looping abstraction.
+    * Element formulations for 1st order:
+      * 8 node hexahedron
+      * 6 node wedge
+      * 5 node pyramid
+      * 4 node tetrahedron
+  * Finite volume stencils for classical (cell-centered) FVM formulation (TPFA, infrastructure support for MPFA, but currently 
+    not implemented)
+  * Inner products for Hybrid FVM formulation (current support for TPFA and quasi-TPFA) 
```

Comment:
I think for release notes specifically there's no point in listing things that aren't yet implemented, so I would just keep it brief
```suggestion
  * Cell-centered Finite Volume method with Two-point Flux Approximation (TPFA)
  * Hybrid Finite Volume method with TPFA and quasi-TPFA inner products
```

Reply:
@klevzoff can you re-suggest this? It is out of date and I can't commit it.

Reply:
Sure, done